serde_v8/error.rs

```diff
@@ -58,6 +58,9 @@ pub enum Error {
 
   #[error("serde_v8 error: can't create slice from resizable ArrayBuffer")]
   ResizableBackingStoreNotSupported,
+
+  #[error("{0}")]
+  Custom(Box<dyn std::error::Error + Send + Sync + 'static>),
```

Comment:
Thinking forward: eventually we'll want to create a deno_core-specific error type, I think.